daml-script/test/src/test-utils/com/daml/lf/engine/script/test/MultiParticipantFixture.scala

```diff
@@ -37,11 +38,12 @@ trait MultiParticipantFixture
 
   private val tmpDir = Files.createTempDirectory("testMultiParticipantFixture")
   private val cantonConfigPath = tmpDir.resolve("participant.config")
+  private val portFile = tmpDir.resolve("portfile")
 
   override protected def afterAll(): Unit = {
     Files.delete(cantonConfigPath)
-    super.afterAll()
-
+    discard(Files.deleteIfExists(portFile))
+    Files.delete(tmpDir)
```

Comment:
We're sure we don't need the logic in `AkkaBeforeAndAfterAll.afterAll`? It seems to shut some things down